### DISCUSSION TITLE
Can we still access ollama :11434 port with docker cuda AIO installs?

### DISCUSSION THREAD

**Discussion by c2h2** [open-webui/open-webui#3954] [General]
I installed this wonderful software with cuda docker AIO install command. but this install only expose 3000 port, not 11434 port which I can still interact with other software like langchain Ollama() stuff.

I have tried launch the docker with this port exposed, but ollama seems not able to accpet non-localhost requests.

Can we still access the 11434 from outside docker?

**Comment by SlavikCA** [comment]
[Here](https://docs.openwebui.com/features) docs said:

> Backend Reverse Proxy Support: Bolster security through direct communication between Open WebUI's backend and Ollama. This key feature eliminates the need to expose Ollama over the local area network (LAN). Requests made to the /ollama/api route from Open WebUI are seamlessly redirected to Ollama from the backend, enhancing overall system security and providing an additional layer of protection.

So, I went to SETTINGS -> ACCOUNT and created API_KEY (not JWT key)

And this request works for me:

```
curl https://my.openweb.ui/ollama/api/generate \
  -H "Content-Type: application/json" \
  -H "Authorization: Bearer sk-d5******" \
    -d '{
  "model": "qwen2.5:32b",
  "stream": false,
  "prompt": "Why is the sky blue?"
}'
```

Here is where API is described:
https://github.com/ollama/ollama/blob/main/docs/api.md

**Comment by o-leary** [comment]
Sadly some integrations appear not to work with this so it can still be useful to expose the port. The only way I've got that to work is via the docker compose install method in the docs here: https://docs.openwebui.com/getting-started/

**Reply by MYanello** [reply]
I found that adding OLLAMA_HOST=0.0.0.0 to the open-webui docker environment settings exposed port 11434 as expected
